pom.xml

```diff
@@ -72,6 +72,7 @@
                 </executions>
                 <configuration>
                     <quiet>true</quiet>
+					<javadocExecutable>${java.home}/bin/javadoc</javadocExecutable>
```

Comment:
Please revert, fixes for local build environment are out of scope

Reply:
Sorry, this is a leftover. It shouldn't have been commited.